tests/run/embedsignatures.pyx

```diff
@@ -185,7 +188,7 @@ __doc__ = ur"""
     f_defexpr4(int x=(Ext.CONST1 + FLAG1) * Ext.CONST2)
 
     >>> print(funcdoc(f_defexpr5))
-    f_defexpr5(int x=4)
+    f_defexpr5(int x=2 + 2)
```

Comment:
This change has pros and cons. If it's really spelled that way in the source file, then it's probably nice to spell it the same way in the docs. However, the expression might stem from a compile time calculated value, which the programmer might not have wanted to leak into the docs. Unclear. I think there might be a case for collapsing the expressions of default arguments and keeping the expressions of annotations.